<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Interface pour VAIMOS (luc.jaulin@ensta-bretagne.fr)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="widgetVaimos" name="widgetVaimos1" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>600</height>
     </size>
    </property>
   </widget>
   <widget class="QScrollBar" name="k0_horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>701</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>50000</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="Echelle_verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>21</width>
      <height>531</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoadFile">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load file</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="Step_verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>16</width>
      <height>531</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="Wind_checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>50</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wind</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="Sail1_checkBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Sail with compass</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="Sail2_checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>90</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sail with motor</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="Text_checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Text</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="Go_checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Go direction</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QScrollBar" name="ks0_verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>260</y>
      <width>20</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="ks1_verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>260</y>
      <width>20</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ks0_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>240</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ks0</string>
    </property>
   </widget>
   <widget class="QLabel" name="ks1_label">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>240</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>ks1</string>
    </property>
   </widget>
   <widget class="QLabel" name="Echelle_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="Step_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="FilteredWindDirMode_checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtered wind mode</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ResetOrigin_checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset origin</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ThickLine_checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Thick line</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="MotorboatMode_checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Motorboat mode</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Recalce_checkBox">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Recalc dist to the line</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>widgetVaimos</class>
   <extends>QWidget</extends>
   <header>widgetVaimos.h</header>
   <container>1</container>
   <slots>
    <signal>SetValue_kmax(int)</signal>
    <signal>SetValue_ks0(int)</signal>
    <signal>SetValue_ks1(int)</signal>
    <slot>VaimosStart()</slot>
    <slot>SetValue_k0(int)</slot>
    <slot>SetValue_ks0(int)</slot>
    <slot>SetValue_ks1(int)</slot>
    <slot>SetValue_Echelle(int)</slot>
    <slot>LoadFile()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
